active/fellowship.md

```diff
@@ -2,30 +2,67 @@
+
+- Chair: TBD
+- Board liaison: TBD
+- Designated manager: TBD
 - Other members:
-  - Andrew Godwin
-  - Frank Wiles
+  - TBD
 
 ## Future membership
 
-Board-managed: the DSF Board appoints members of this WG.
+Board-managed: the DSF Board appoints members of this WG, including specific roles (chair, Fellow manager).
 
-Membership is open to volunteers, subject to board approval. [Contact the board](https://www.djangoproject.com/contact/foundation/) to express interest.
+The Board will select membership with some high-level goals in mind:
+
+- **Diversity**: the WG membership should be diverse, as representative of the global Django community as possible within the confines of a small team.
+- **Management experience**: WG members, _particularly_ the designated manager, should have some engineering management experience, or other relevant experience (i.e. experience with fellowship programs at other NPOs, etc). This isn't a hard requirement and can easily be outweighed by the other things the potential member brings to the table, but it is something the board will generally look for.
+- **Django community experience**: WG members generally speaking should have been members of the Django community for some time. Once again this isn't a hard requirement, but one of the duties of the WG is to help Fellows navigate the community, if needed, so people with experience there are nice to have on the board.
+
+Membership is open to volunteers, subject to board approval. [Contact the board](https://www.djangoproject.com/contact/foundation/) to express interest. Please include a brief explanation of how your background aligns with the goals above.
+
+Members serve a 1-year term, April 1st to March 31st, in alignment with the Fellow contracts. At the end of this term, they need to opt into staying involved to keep being
+a member of the group. If any member wishes to leave the group before the end of their term, they can do so without a vote.
 
 ## Budget
 
-The DSF Board sets the Fellowship compensation, but the WG then has the power to select who fills the position.
+- The Board sets the Fellowship compensation (which is adjusted yearly, automatically, to match CPI inflation).
+- The Board establishes a yearly Travel & Expenses budget, which Fellows may spend against (with management approval). This is set to a level to allow Fellows to attend at least one DjangoCon.
+  - Expenses beyond this yearly budget may (probably will!) be approved, but would require approval by the board to increase the budget.
 
 ## Comms
 
-TBD
+The Fellowship-WG has a private mailing list, and uses the DSF Slack. Fellows themselves generally work in the open, on the Django Forum and other public spaces.
 
 ## Reporting
 
-TBD
+The Fellowship WG is required to submit regular reports on the impact and efficacy of the Fellow program.
```

Comment:
Across the document there are usages of "Fellowship WG" and "Fellowship-WG", is there a preferred spelling and should we stick to that one?